modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc

```diff
@@ -110,34 +110,30 @@ from kfp import dsl
 )
 
 
-def ray_fn(openshift_server: str, openshift_token: str):
+def ray_fn():
    import ray <1>
-   from codeflare_sdk import Cluster, ClusterConfiguration, TokenAuthentication, generate_cert
+   import time <2>
+   from codeflare_sdk import Cluster, ClusterConfiguration, generate_cert <3>
 
 
-   auth = TokenAuthentication( <2>
-       token=openshift_token, server=openshift_server
-   )
-   auth_return = auth.login()
-
-
-   cluster = Cluster( <3>
+   cluster = Cluster( <4>
        ClusterConfiguration(
+           namespace="my_project", <5>
```

Comment:
Can be removed

Reply:
Updated instead to indicate that it's optional, as agreed in Slack.